fs/rc/rcserver/rcserver.go

```diff
@@ -265,6 +265,10 @@ func (s *Server) handlePost(w http.ResponseWriter, r *http.Request, path string)
 		in["_request"] = r
 	}
 
+	if call.NeedsResponse {
+		in["_response"] = &w
```

Comment:
The `http.ResponseWriter` is an interface so is safe to pass by value here so just

    in["_response"] = w

The signature of `GetHTTPResponseWriter` will need to change to

    func (p Params) GetHTTPResponseWriter() (*http.ResponseWriter, error) {

An interface in Go is already contains a pointer. Effectively it is a very small structure with two pointers, one to the actual data and one to a description of the data.